test/test-parse-class.cpp

```diff
@@ -1078,6 +1078,20 @@ TEST(test_parse, class_statement_as_with_statement_body_is_disallowed) {
             offsets_matcher(&code, strlen(u8"with (obj) "), u8"class"))));
   }
 }
+
+TEST(test_parse, async_static_method_is_disallowed) {
+  {
+    spy_visitor v;
+    padded_string code(u8"class C { async static m() {} }"_sv);
```

Comment:
Verify that error recovery works by checking visits. In particular, a class property called `m` should be declared, and its body should be visited, and `await` should be treated as an operator inside the body.